docs/misc/commands.md

```diff
@@ -0,0 +1,122 @@
+---
+sidebar-position: 7
+---
+# Commands
+Commands are very useful in vanilla Minecraft and even more so with mods. They are the quickest way to access and modify data.
+Commands allow to:
+- Use permission levels.
+- Access to anything in your mod.
+- Allow use anywhere on the map, generally without specific items or other physical objects.
+## Creating a command
+To create a command, you need to create a new class and put this code:
+```java
+public static void register(CommandDispatcher<CommandSourceStack> dispatcher) {
+    dispatcher.register (
+        Commands.literal("commandname") // The name of your command
```

Comment:
Why does it start with a literal?

Reply:
But that's not explained. I could easily pass in an argument, but why should I start with a `literal`. It is not necessary for `register`, but it is the best practice to do so.